src/napari/layers/tracks/tracks.py

```diff
@@ -93,6 +93,9 @@ class Tracks(Layer):
         Length of the positive (backward in time) tails in units of time.
     tail_width : float
         Width of the track tails in pixels.
+    hide_finished_tracks : bool
```

Comment:
I'm sure you put it here because it's more semantically related to the tail, but the layer docstrings are alphabetically sorted by attribute name, so we should move this up to the correct spot. This should also fix your test failure.